include/zephyr/sys/spsc_pbuf.h

```diff
@@ -31,11 +31,25 @@ extern "C" {
 #define SPSC_PBUF_UTILIZATION_OFFSET 8
 
 /**@} */
+#ifdef DCACHE
```

Comment:
`#ifdef CONFIG_DCACHE`

uhm, we should probably make `DCACHE_LINE_*` depending on `DCACHE` instead of having this additional check.

Reply:
@carlocaione I reworked it a bit. Can you take another look?

Reply:
Well, I took a look and the usage of `CONFIG_DCACHE` is messy (it was introduced by a later and urelated commit). I need to clean that up but before that can you please:
- Remove dependency on `CONFIG_DCACHE` (sorry about that)
- Just use `CONFIG_DCACHE_LINE_SIZE != 0`? (isn't that implying that `CONFIG_DCACHE_LINE_SIZE` is defined?)

Reply:
done